.overcommit.yml

```diff
@@ -9,6 +9,8 @@ PreCommit:
 
   HardTabs:
     enabled: true
+    exclude:
+      - 'lib/haml_lint/linter/README.md'
```

Comment:
I don't think you can do that for overcommit. It makes sense to me to exclude it.